src/Forside/Sammenligningspanel/MaskertSykefraværprosentpanel/DataKanIkkeVisesPanel/DataKanIkkeVisesPanel.less

```diff
@@ -2,10 +2,9 @@
 
 .data-kan-ikke-vises-panel {
     display: flex;
-    border-bottom: 0.05rem solid @navBlaLighten60;
     align-items: center;
     padding-bottom: 1rem;
-
+    max-width: 25%;
```

Comment:
Kan vi sette `width: 25%` i stedet for `max-width`? Da ser det mindre rotete ut med små navn. F.eks. Norge-boksen blir litt liten nå.